include/NaluCommNeighbors.hpp

```diff
@@ -122,7 +122,6 @@ class CommNeighbors {
   std::vector<unsigned char> m_recv_data;
   std::vector<int> m_send_procs;
   std::vector<int> m_recv_procs;
```

Comment:
If you use MPI neighborhood collectives, you don't actually need to keep the send and receive process ranks.  MPI keeps them for you.  MPI has functions for querying a topology, so you can get this information if you need it.

Reply:
Yes, that's true... If I do want to allow people to get them, I think hanging onto the vectors is a better solution. I know MPI provides functions for getting the neighbor-counts and the neighbors, but then I have to make the counts call, allocate the vector, then make the second call... I'm just not sure yet about the use-case for a caller needing them. That probably means I should remove the access entirely until the need becomes clear.

Reply:
I agree that removing access for now is a good idea :-) .